index.html

```diff
@@ -96,7 +96,7 @@
       <div class="grid">
         <div class="u-1-2">
           <p>
-            Like what you hear? Visit the documentation and get started today. <br>
+            Like what you hear? Visit the documentation and get started today. <br><br>
```

Comment:
I believe that when I first started (before the rows were set to columns on mobile) the two buttons (line 108 and 115) were overlapping with the paragraph beside it I added that so that it would fit a bit better, but when I started working on switching the rows to columns, that became useless. I guess I just for got to remove it.